src/ui/components/questions/questionsubmissioncomponent.js

```diff
@@ -30,43 +31,62 @@ const DEFAULT_CONFIG = {
    * An optional label to use for the e-mail address input
    * @type {string}
    */
-  'emailLabel': 'Email',
+  'emailLabel': TranslationFlagger.flag({
+    phrase: 'Email',
+    context: 'Label for an input field'
+  }),
 
   /**
    * An optional label to use for the name input
    * @type {string}
    */
-  'nameLabel': 'Name',
+  'nameLabel': TranslationFlagger.flag({
+    phrase: 'Name',
+    context: 'Label for an input field'
+  }),
 
   /**
    * An optional label to use for the question
    * @type {string}
    */
-  'questionLabel': 'Question',
+  'questionLabel': TranslationFlagger.flag({
+    phrase: 'Question',
+    context: 'Label for an input field'
```

Comment:
Is `Label for an input field` the sort of context we give in Alpha?

Reply:
The original PO file I found didn't have any good examples of this 🙈, I dug some more, and a similar example is: `Labels the minute value of the time that a business opens on the day of the week provided as an argument.` I can update these to align more closely with that example.